port/common/omrstr.c

```diff
@@ -2133,6 +2133,86 @@ omrstr_subst_time(struct OMRPortLibrary *portLibrary, char *buf, uint32_t bufLen
 	return count;
 }
 
+/**
+ * Retrieve the offset of local time from UTC and the current time zone name. If the current time zone could not be determined, this function will
+ * not modify its outputs.
+ *
+ *
+ * @param[in] portLibrary the port library
+ * @param[out] hourOffset the hour offset from UTC+0, which may be negative
+ * @param[out] minuteOffset the minute offset from UTC+0, in the range [0, 59]
+ * @param[out] buf the string buffer to be filled with the time zone name
+ * @param[in] bufLen the buffer length
```

Comment:
See name suggestions above. 

A caller might not be interested in the timezone name, it should be acceptable for this parameter to be `NULL` in that case. Edit: it seems the code behaves that way: it should be documented here.